src/app/components/popup.ts

```diff
@@ -0,0 +1,45 @@
+export function renderPopup(popupContentHTML: string, closeBtnContent: string) {
```

Comment:
Will we use different content for close button? Isn't it just the same close button for every view?
I am not sure if we should pass popupContentHTML as string, it can be then again problem with adding event listeners. Maybe can we pass here HTMLElement?

Reply:
Thanks to the closebtncontent we can set button value to 'save' or 'close' so its more reusable. We can set default text to it. I think we can left it for now. To the pop up content you are right. There will be problems with events in the future